### PR TITLE
Add DigitalTwins Generated Code and Tests

### PR DESCRIPTION
**Pull request by Youri970410**
## Description

Generate DigitalTwins code base on Version '2020-10-31' and test all cmdlets.

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/DigitalTwins/docs/New-AzDigitalTwinsCheckNameRequestObject.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+```
+New-AzDigitalTwinsCheckNameRequestObject -Name <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create a in-memory object for CheckNameRequest
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
```
Comment (dolauli) [inline src/DigitalTwins/docs/New-AzDigitalTwinsCheckNameRequestObject.md:24]: Please fill in an example here

File: src/DigitalTwins/docs/New-AzDigitalTwinsDigitalTwinsIdentityObject.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+New-AzDigitalTwinsDigitalTwinsIdentityObject [-EndpointName <String>] [-Id <String>] [-Location <String>]
+ [-ResourceGroupName <String>] [-ResourceName <String>] [-SubscriptionId <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create a in-memory object for DigitalTwinsIdentity
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
```
Comment (dolauli) [inline src/DigitalTwins/docs/New-AzDigitalTwinsDigitalTwinsIdentityObject.md:25]: Please fill in an example here

File: src/DigitalTwins/docs/Remove-AzDigitalTwinsEndpoint.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> Remove-AzDigitalTwinsEndpoint -ResourceGroupName youritemp -EndpointName youriEHEndpoint -ResourceName youriDigitalTwinsTest
+
+```
+
+{{ Add description here }}
+
+### Example 2: {{ Add title here }}
```
Comment (dolauli) [inline src/DigitalTwins/docs/Remove-AzDigitalTwinsEndpoint.md:41]: Clean example templates

File: src/DigitalTwins/examples/New-AzDigitalTwinsCheckNameRequestObject.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
```
Comment (dolauli) [inline src/DigitalTwins/examples/New-AzDigitalTwinsCheckNameRequestObject.md:5]: Fill in an example

File: src/DigitalTwins/examples/New-AzDigitalTwinsDigitalTwinsIdentityObject.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
+
+{{ Add output here }}
+```
+
+{{ Add description here }}
```
Comment (dolauli) [inline src/DigitalTwins/examples/New-AzDigitalTwinsDigitalTwinsIdentityObject.md:8]: Fill in an example

File: src/DigitalTwins/examples/Remove-AzDigitalTwinsEndpoint.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> Remove-AzDigitalTwinsEndpoint -ResourceGroupName youritemp -EndpointName youriEHEndpoint -ResourceName youriDigitalTwinsTest
+
+```
+
+{{ Add description here }}
+
```
Comment (dolauli) [inline src/DigitalTwins/examples/Remove-AzDigitalTwinsEndpoint.md:8]: Clean example templates

**Review by dolauli — CHANGES_REQUESTED** [review]
Some changes are required.

File: src/DigitalTwins/docs/New-AzDigitalTwinsCheckNameRequestObject.md (hunk 1 above)
Comment (Youri970410) [reply src/DigitalTwins/docs/New-AzDigitalTwinsCheckNameRequestObject.md:24]: Got it,thanks.

File: src/DigitalTwins/docs/New-AzDigitalTwinsDigitalTwinsIdentityObject.md (hunk 2 above)
Comment (Youri970410) [reply src/DigitalTwins/docs/New-AzDigitalTwinsDigitalTwinsIdentityObject.md:25]: Got it,thanks.

File: src/DigitalTwins/examples/New-AzDigitalTwinsCheckNameRequestObject.md (hunk 4 above)
Comment (Youri970410) [reply src/DigitalTwins/examples/New-AzDigitalTwinsCheckNameRequestObject.md:5]: Got it,thanks.

File: src/DigitalTwins/examples/New-AzDigitalTwinsDigitalTwinsIdentityObject.md (hunk 5 above)
Comment (Youri970410) [reply src/DigitalTwins/examples/New-AzDigitalTwinsDigitalTwinsIdentityObject.md:8]: Got it,thanks.

File: src/DigitalTwins/examples/Remove-AzDigitalTwinsEndpoint.md (hunk 6 above)
Comment (Youri970410) [reply src/DigitalTwins/examples/Remove-AzDigitalTwinsEndpoint.md:8]: Got it,thanks.

File: src/DigitalTwins/docs/Remove-AzDigitalTwinsEndpoint.md (hunk 3 above)
Comment (Youri970410) [reply src/DigitalTwins/docs/Remove-AzDigitalTwinsEndpoint.md:41]: Got it,thanks.

**Review by dolauli — APPROVED** [review]

**Pull request merged**